treeherder/settings/base.py

```diff
@@ -219,6 +225,14 @@
         },
         'relative': True
     },
+    'fetch-missing-push-logs-every-minute': {
+        'task': 'fetch-missing-push-logs',
+        'schedule': timedelta(minutes=1),
+        'options': {
+            "queue": "pushlog"
+        },
```

Comment:
Whilst you are here, could you move "options" after "relative", to be consistent with the others? (Or at least make them all the same, whatever order that is)